modules/ROOT/pages/index.adoc

```diff
@@ -25,6 +25,40 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
 
 If you are already using a PaaS solution, MuleSoft recommends deploying Runtime Fabric in parallel with your PaaS. This enables you to take advantage of the complete benefits of Anypoint Platform.
 
+== Shared Responsibility for On-Premises Runtime Fabric
+
+The successful operation of on-premises Runtime Fabric instances is a shared responsibility. It is critical to understand which areas you must manage and which areas are managed by MuleSoft. Without the required infrastructure and proper maintenance, Runtime Fabric will not operate successfully.
+
+The following image illustrates MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
+
+image::sharedresponsibility.png[Runtime Fabric Shared Responsiblities]
+
+* MuleSoft responsibility - MuleSoft manages the RTF control plane and is responsible for the delivered components, Runtime Fabric appliance, Runtime Fabric agent, Mule runtime engine, and other dependencies for Mule appications.
+
+* Customer responsibility - Customers are responsible for provisioning, configuring, and managing the infrastructure required for Runtime Fabric and Runtime Fabric clusters.  
++
+** Infrastructure includes:
++
+*** VM resources (CPU, Memory)
++
+*** Disk performance and capacity
++
+*** Operating systems and kernel patching
++
+*** Network ports
++
+*** Syncing system time across all VMs
++
+** For infrastructure provisioning and management, you will need the assistance of your organization’s:
++
+*** DevOps team to provision and manage the infrastructure
++
+*** Network team to whitelist ports and configure proxy settings
++
+*** Security team to review for compliance and obtain security certificates
+
+Refer to xref:install-prereqs.adoc[Runtime Fabric Installation Prerequisites] for additional information.
```

Comment:
Per our new linking policies, only need-to-know-right-this-minute xrefs appear in main narrative and all others go into a ==See Also subtopic at the end of the topic set.
Also per policy, if we provide a reason for the xref, it should be more robust than "for additional information": for information about what, specifically, and to what user purpose?